src/app/page.tsx

```diff
@@ -48,6 +47,7 @@ export default function Home() {
         singleBlog={false}
       />
       <ArticleList />
+      <Teacher />
```

Comment:
Keep the teacher section where it was, add the subscribe section here. Similar to the section under articles and do the same for neurodivergent mates like this ->
<img width="1399" alt="image" src="https://github.com/NeurodiversityAcademy/neurostudy-portal/assets/150103212/1ceeb9c3-e2a1-4372-9199-933644b97a20">